docs/patterns/components/Accordion/index.js

```diff
@@ -0,0 +1,172 @@
+            Accordion #1
+          </AccordionTrigger>
+          <AccordionContent>Here is some content</AccordionContent>
+        </Accordion>
+        <Accordion>
+          <AccordionTrigger heading={{ level: 4 }}>
+            Accordion #2
+          </AccordionTrigger>
+          <AccordionContent>Here is some content</AccordionContent>
+        </Accordion>
+        <Accordion>
+          <AccordionTrigger heading={{ level: 4 }}>
+            Accordion #3
+          </AccordionTrigger>
+          <AccordionContent>Here is some content</AccordionContent>
+        </Accordion>
+        <Code role="region" tabIndex={0}>
+          {`<Accordion>
+    <AccordionTrigger>Accordion #3</AccordionTrigger>
+    <AccordionContent>Here is some content</AccordionContent>
+</Accordion>`}
+        </Code>
+        <h3>Controlled</h3>
+        <p>
+          If you need to manually control the open/closed state of the panel,
+          you can do this by setting the open prop and handling changes with
+          onToggle.
+        </p>
+        <ControlledAccordion label="Accordion #1" heading={{ level: 4 }} />
+        <ControlledAccordion label="Accordion #2" heading={{ level: 4 }} />
+        <ControlledAccordion label="Accordion #3" heading={{ level: 4 }} />
+        <Code role="region" tabIndex={0}>
+          {`import React, { useEffect, useState } from 'react';
+  import {
+  Accordion,
+  AccordionTrigger,
+  AccordionContent,
+} from '@deque/cauldron-react/';
+
+const ControlledAccordion = ({ label }) => {
+  const [open, setIsOpen] = useState(false);
+
+  return (
+    <Accordion open={open} onToggle={() => setIsOpen(!open)}>
+    <Accordion>
+    <AccordionTrigger heading={{ level: 4 }}>
+      {label}
+    </AccordionTrigger>
+    <AccordionContent>Here is some content</AccordionContent>
+  </Accordion>
+  );
+};`}
+        </Code>
+      </div>
+      <div className="Demo-props">
+        <h2>Props</h2>
+        <h3>
+          <code>Accordion</code>
+        </h3>
+        <p>
+          The <code>Accordion</code> supports all the props from the{' '}
+          <Link href="/components/ExpandCollapsePanel">
+            ExpandCollapsePanelProps
+          </Link>
+          including/in addition to the following:
+        </p>
+
+        <PropDocs
+          docs={{
+            className,
+            children,
+            open: {
+              type: 'boolean',
+              description: 'Initial collapsed state of ExpandCollapsePanel',
+              default: 'false'
+            },
+            onToggle: {
+              type: '(e: React.MouseEvent<HTMLButtonElement>) => void',
+              description:
+                'onToggle handler for the panel. The original event object will be passed.',
+              default: 'function () {}'
+            }
+          }}
+        />
+
+        <h3>
+          <code>AccordionTrigger</code>
+        </h3>
+        <p>
+          The <code>AccordionTrigger </code> extends
```

Comment:
```suggestion
          The <code>AccordionTrigger </code> extends{' '}
```

Isn't React great?

Reply:
Hah! Yeah, the spacing on the prop doc pages was getting really weird sometimes. I kept removing things, then sometimes prettier would add back in, sometimes I have no idea how it happened 😂